### PR TITLE
new autodisco landing page

### PR DESCRIPTION
**Pull request by cswatt**
revamp the autodiscovery landing page

https://docs-staging.datadoghq.com/cswatt/autodisco-landing/agent/autodiscovery

### CODE REVIEW

File: content/en/agent/autodiscovery/_index.md [hunk 1]
```diff
@@ -22,69 +22,51 @@ further_reading:
...
+Add the following configuration block in the `datadog.yaml` [configuration file][1]. 
 
-{{< tabs >}}
-{{% tab "Agent" %}}
-
-To enable Autodiscovery over containers within Kubernetes, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
-
-```yaml
+```
 listeners:
```
Comment (cohenyair) [inline content/en/agent/autodiscovery/_index.md]: I don't think that you need explicit configuration to enable autodiscovery. AFAIK it is enabled by default when deployed on Kubernentes, see for example the blog post that shows how to monitor EKS that doesn't mention these options: https://www.datadoghq.com/blog/eks-monitoring-datadog/

File: content/en/agent/autodiscovery/_index.md [hunk 2]
```diff
@@ -22,69 +22,51 @@ further_reading:
...
+2. Create [integration-specific configuration templates](#integration-templates) for each service you wish to monitor. Note that Datadog provides auto-configuration templates for [some common containerized services][3], including Apache and Redis.
 
-2. **Apply an integration template to a specific container**: Unlike in a traditional Agent setup, the Agent doesn't run all checks all the time; it decides which checks to enable by inspecting all containers running on the same host as the Agent and the corresponding loaded integration templates. The Agent then watches for Kubernetes/Docker events&mdash;container creation, destruction, starts, and stops&mdash;and enables, disables, and regenerates static check configurations on such events. As the Agent inspects each running container, it checks if the container matches any of the [Autodiscovery container identifiers][3] from any loaded integration templates. For each match, the Agent generates a static check configuration by substituting the [Template Variables][10] with the matching container's specific values. Then it enables the check using the static configuration.
+### Enable Autodiscovery 
 
-## How to set it up
```
Comment (cohenyair) [inline content/en/agent/autodiscovery/_index.md]: Can we focus on showing examples of autodiscovery annotations for a K8 pod/deployment or ECS Task first? The example should explain how to use autodiscovery and how the parameters work

File: content/en/agent/autodiscovery/_index.md [hunk 3]
```diff
@@ -129,19 +115,20 @@ ECS_FARGATE=true
 {{% /tab %}}
 {{< /tabs >}}
 
+### Integration Templates
```
Comment (cohenyair) [inline content/en/agent/autodiscovery/_index.md]: Can you add an example and an explanation to it?

File: content/en/agent/autodiscovery/_index.md [hunk 4]
```diff
@@ -22,69 +22,51 @@ further_reading:
...
-This page covers Autodiscovery with Agent 6 only, [refer to the dedicated documentation to setup Autodiscovery with Agent 5][1]
+When you are monitoring a containerized infrastructure, one challenge that arises is that containers can shift from host to host. The dynamic nature of containerized systems makes them difficult to manually monitor.
 
-Here's a 5-minute video for a bird's eye view of Datadog Agent v6 Autodiscovery functionality.
+To solve this issue, you can use Datadog’s Autodiscovery feature to automatically identify the services running on a specific container and gather data from those services. Whenever a container starts, the Datadog Agent identifies which services are running on this new container, looks for the corresponding monitoring configuration, and starts to collect metrics.
 
-{{< wistia mlxx0j6txw >}}
+Autodiscovery lets you define configuration templates for Agent checks and specify which containers each checks should apply to. 
 
-## How it Works
```
Comment (cohenyair) [inline content/en/agent/autodiscovery/_index.md]: A diagram can really add value to simplify the flow IMO

File: content/en/agent/autodiscovery/_index.md [hunk 5]
```diff
@@ -129,19 +115,20 @@ ECS_FARGATE=true
...
 
+### Integration Templates
+
+Once Autodiscovery is enabled, the Datadog Agent automatically attempts Autodiscovery [for a number of services][3], including Apache and Redis, based on default Autodiscovery configuration files.
+
+To use Autodiscovery with other services, define templates for the services you wish to monitor. See the [Autodiscovery Integration Templates][4] documentation for further details.
+
+
+
 ## Further Reading
```
Comment (cohenyair) [inline content/en/agent/autodiscovery/_index.md]: Can you add a section of the different checks that use/relate to Autodiscovery? Specifically I'd like to clarify when to use Cluster-checks, Agent-checks and others (i.e. end-point checks). We can also refer to the pages that explain them for further reading

File: content/en/agent/autodiscovery/_index.md (hunk 1 above)
Comment (cswatt) [reply content/en/agent/autodiscovery/_index.md]: added a note that AD is enabled by default on K8s

File: content/en/agent/autodiscovery/_index.md (hunk 2 above)
Comment (cswatt) [reply content/en/agent/autodiscovery/_index.md]: I think putting an example of that so far up on the page (before we talk about enabling autodiscovery) is confusing, and would be more helpful added later down on the page.

File: content/en/agent/autodiscovery/_index.md (hunk 3 above)
Comment (cswatt) [reply content/en/agent/autodiscovery/_index.md]: added quick redis example below, users should follow link to Integrations Templates page to read more.

**Pull request merged**